web/pdf_viewer.css

```
@@ -25,6 +25,7 @@
   --page-border-image: url(images/shadow.png) 9 9 repeat;
   --spreadHorizontalWrapped-margin-LR: -3.5px;
   --scale-factor: 1;
+  --loading-delay: 400ms;
```

How about the following name instead, to make it slightly more clear what it's being used for?
```suggestion
  --loading-icon-delay: 400ms;
```